src/modules/timezones/entities/timezone.entity.ts

```diff
@@ -0,0 +1,17 @@
+import { Entity, Column, PrimaryGeneratedColumn } from 'typeorm';
+import { AbstractBaseEntity } from '../../../entities/base.entity'; // Assume you have a BaseEntity defined
+
+@Entity()
+export class Timezone extends AbstractBaseEntity {
+  @PrimaryGeneratedColumn('uuid')
```

Comment:
Extending the AbstractBaseEntity provides this already, kindly remove the id property definition

Reply:
Done